yellowbrick/features/pca.py

```diff
@@ -64,6 +65,9 @@ class PCADecomposition(MultiFeatureVisualizer):
 
     color : list or tuple of colors, default: None
         Specify the colors for each individual class.
+        
+    legend_labels : list or tuple of values for an optional legend that lists 
```

Comment:
I'm thinking it would be best to use `classes` here instead of `legend_labels`; in addition to not introducing new params into YB, this would also be more analogous to what we do in the `DataVisualizers` (though I'm okay with keeping this as `MultiFeatureVisualizer`)